yellowbrick/classifier/class_balance.py

```diff
@@ -25,6 +25,8 @@
 
 from .base import ClassificationScoreVisualizer
 
+from ..style.palettes import color_palette
```

Comment:
I think that lines 28 and 29 are remainders from previous work; but are causing the tests to fail:

```
________________________________ pyflakes-check ________________________________
/home/travis/build/DistrictDataLabs/yellowbrick/yellowbrick/classifier/class_balance.py:28: UnusedImport
'..style.palettes.color_palette' imported but unused
```

Would you mind removing these lines, I recommend the following:

```
$ git checkout yellowbrick/classifier/class_balance.py
```

Reply:
@bbengfort line 29 is empty - did you mean just remove line 28?

Reply:
Only line 28 is causing the tests to fail, but I do recommend the removal of both lines so that no changes are made to this file, which has no impact on your PR at all. Again, you can reset this file with:

```
$ git checkout yellowbrick/classifier/class_balance.py 
```